tests/testsuite/auto_complete/basic/mod.rs

```diff
@@ -0,0 +1,94 @@
+use cargo_test_support::cargo_test;
+use cargo_test_support::current_dir;
+use cargo_test_support::Project;
+use snapbox::assert_data_eq;
+
+#[cargo_test]
+fn bash() {
+    let project = Project::from_template(current_dir!().join("in"));
+    let project_root = project.root();
+    let cwd = &project_root;
+
+    // Change current directory to the project root
+    std::env::set_current_dir(cwd).unwrap();
+
+    let input = "cargo \t\t";
+    let expected = snapbox::str![
+        "% 
+--version          --help             check              install            read-manifest      update
+--list             -V                 clean              locate-project     remove             vendor
+--explain          -v                 config             login              report             verify-project
+--verbose          -q                 doc                logout             run                version
+--quiet            -C                 fetch              metadata           rustc              yank
+--color            -Z                 fix                new                rustdoc            
+--locked           -h                 generate-lockfile  owner              search             
+--offline          add                help               package            test               
+--frozen           bench              info               pkgid              tree               
+--config           build              init               publish            uninstall          "
+    ];
+    let actual = super::common::complete(input, "bash");
+    assert_data_eq!(actual, expected);
+
+    let input = "cargo build --\t\t";
```

Comment:
We probably only need one per shell.  Our focus for these tests should be on the completions being registered.  For everything else, we should either directly test custom completers or rely on `clap_complete`s tests

Reply:
In that situation, do we need to have this setup for many `.rs` files?